src/components/contracts/utils/index.js

```diff
@@ -176,3 +176,17 @@ export const defaultContract = (fieldValues) => ({
   fieldValues,
   children: null,
 });
+
+export const previousContract = (previousContract) => {
```

Comment:
looks more like a clone than "a previous contract"

Reply:
Here, I set the Id to 0 because I want the form to know that we want to create a new contract not the edit an existing one.